content/pt-br/blog/_posts/2023-03-10-image-registry-change.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: blog
+title: "Redirecionamento do registry de imagens k8s.gcr.io para registry.k8s.io - O que você precisa saber"
+date: 2023-03-10T17:00:00.000Z
+slug: image-registry-redirect
+---
+
+**Autores**: Bob Killen (Google), Davanum Srinivas (AWS), Chris Short (AWS), Frederico Muñoz (SAS
+Institute), Tim Bannister (The Scale Factory), Ricky Sadowski (AWS), Grace Nguyen (Expo), Mahamed
+Ali (Rackspace Technology), Mars Toktonaliev (independent), Laura Santamaria (Dell), Kat Cosgrove
+(Dell)
+
+
+Na segunda-feira, dia 20 de Março, o registry de imagens k8s.gcr.io [vai ser redirecionado para o registry da comunidade](https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/),**registry.k8s.io**.
+
+
+## TL;DR: O que você precisa saber sobre essa mudança
+- Na segunda-feira, dia 20 de Março, o tráfego do registry antigo k8s.gcr.io vai ser redirecionado para
+registry.k8s.io no eventual objetivo de encerramento do k8s.gcr.io.
+- Se você está executando em um ambiente restrito, e aplica políticas rígidas de acesso a nomes de domínios
+ou endereços IPs limitado a k8s.gcr.io, **o pull de imagens não vai funcionar** depois que o k8s.gcr.io começar
+a redirecionar para o novo registry.
+- Um pequeno subconjunto de clients que não seguem o padrão não lidam com redirecionamentos HTTP, nesse caso eles
+precisam ser apontados diretamente para registry.k8s.io.
+- O redirecionamento é um paliativo para ajudar os usuários a fazer essa trocy. O registro obsoleto k8s.gcr.io será desativado em algum momento. **Por isso, atualize seus manifestos o mais rápido possível para apontar para registry.k8s.io**.
+- Se você mantém seu próprio registry de imagens, voc6e pode copiar as imagens que você precisa para reduzir o tráfego
+ao registry da comunidade.
+
+Se você acha que vai ser impactado, ou gostaria de saber mais sobre essa mudança, continue a leitura.
+
+## Como eu posso checar se eu vou ser afetado?
+Para testar a conectividade ao registry.k8s.io (e saber se consegue fazer o pull de imagens), pode executar o
+comando abaixo em um namespace da sua escolha:
+
+```
+kubectl run hello-world -ti --rm --image=registry.k8s.io/busybox:latest --restart=Never -- date
+```
+
+Quando executar o comando acima, é esperada a seguinte saída caso tudo esteja funcionando corretamente:
+
+```
+$ kubectl run hello-world -ti --rm --image=registry.k8s.io/busybox:latest --restart=Never -- date
+Fri Feb 31 07:07:07 UTC 2023
+pod "hello-world" deleted
+```
+
+## Quais erros são esperados caso eu seja afetado?
+Os erros podem depender do tipo de container runtime que está usando, e de qual endpoint você está 
+roteando para, mas devem ser erros como `ErrImagePull`, `ImagePullBackOff`, ou falha na criação do
+container com o aviso `FailedCreatePodSandBox`.
+
+Abaixo um exemplo de uma mensagem de erro mostrando um deployment de proxy falhando devido um certificado desconhecido:
```

Comment:
Suggestion: "instalação por trás de um proxy". Proxied deployment gives me the idea of a deployment sitting behind a proxy, not a deployment of a proxy.